operator/builtin/input/file/testdata/include_file_name_lower.yaml

```diff
@@ -0,0 +1,4 @@
+type: file_input
+include:
+  - one.log
+include_file_name: true
```

Comment:
We might as well include some `false` value tests as well.

Also, I think it would be good to have some negative tests, just to capture expected behavior for invalid inputs.